kythe/go/extractors/cmd/bazel/bazel_go_extractor/bazel_go_extractor.go

```diff
@@ -207,12 +223,23 @@ func parseCompileArgs(args []string) *compileArgs {
 		switch flag {
 		case "dep":
 			c.deps = append(c.deps, arg)
-		case "importmap", "arc":
+		case "importmap":
 			// Only record the mappings that change something.
 			ps := strings.SplitN(arg, "=", 2)
 			if len(ps) == 2 && ps[0] != ps[1] {
 				c.importMap[ps[0]] = ps[1]
 			}
+		case "arc":
+			// Example -arc arg:
+			//   "github.com/google/subcommands=github.com/google/subcommands=bazel-out/k8-fastbuild/bin/external/com_github_google_subcommands/linux_amd64_stripped/go_default_library%/github.com/google/subcommands.a=",
+			//
+			// Only record the mappings that change something.
+			ps := strings.SplitN(strings.TrimSuffix(arg, "="), "=", 3)
+			if len(ps) == 3 {
```

Comment:
Maybe `if len(ps) == 3 && ps[1] != ps[2] {` if you want to be really safe.

Reply:
From what I can tell, `ps[2]` should always be a path to a `.a` file, while `ps[0]` should always be an import path.

The `-arc` flag comes from:
https://github.com/bazelbuild/rules_go/blob/8ea79bbd5e6ea09dc611c245d1dc09ef7ab7118a/go/tools/builders/compile.go#L43